test/index.js

```diff
@@ -22,12 +20,6 @@ import {
   ShowMessageRequest
 } from 'vscode-languageserver-protocol/node.js'
 
-test('exports', (t) => {
```

Comment:
I'm a bit cautious on removing tests.
There should still be export, even if they are partiallytested elsewhere?

Reply:
> I'm a bit cautious on removing tests.

I think that’s a good quality. :)

---

IMHO this test is a bit superfluous. It literally asserts the export is a function. All other tests test the behaviour of this function (fully, not partially).

Still, if you have strong feelings about this, I can restore it.

Reply:
I believe that the reason for something like this in the tests was because there was an export that was no longer needed.

A better way to test the API surface, which somewhat makes sure that we don’t export things we don’t need, and we do not accidentally remove things that we do need, would be something along the lines of:

```js
const api = await import('unified-language-server')
const keys = Object.keys(api).sort()
assert.deepEqual(keys, ['something', 'somethingElse'])
```